src/table/utils/styling-utils.js

```diff
@@ -73,12 +72,14 @@ export function getHeaderStyle(layout, theme) {
   const headerStyle = getBaseStyling(header, 'header', theme);
   headerStyle.borderWidth = '1px 1px 1px 0px';
 
-  // When the table background color from the sense theme is transparent,
+  // To avoid seeing the table body through the table head:
+  // - When the table background color from the sense theme is transparent,
   // there is a header background color depending on the header font color
-  // to avoid seeing the table body through the table head.
+  // - When the table background color from the sense theme has opacity,
+  // removing that.
   const headerBackgroundColor = isDarkColor(headerStyle.color) ? '#FAFAFA' : '#323232';
-  headerStyle.backgroundColor =
-    theme.table.backgroundColor === 'transparent' ? headerBackgroundColor : theme.table.backgroundColor;
+  const RGB = removeOpacity(theme.table.backgroundColor);
+  headerStyle.backgroundColor = theme.table.isBackgroundTransparentColor ? headerBackgroundColor : RGB;
```

Comment:
No need to run removeOpacity if it is not needed
```suggestion
  headerStyle.backgroundColor = theme.table.isBackgroundTransparentColor ? headerBackgroundColor : removeOpacity(theme.table.backgroundColor);
```

Reply:
wait shouldn't it be the other way around?

Reply:
No, fixed

Reply:
Looked at the old PR and it makes sense